test/functorch/test_control_flow.py

```diff
@@ -1557,6 +1558,7 @@ def f(x, y):
         self.assertEqual(graph(x, torch.tensor(True)), f(x, torch.tensor(True)))
 
     @skipIfTorchDynamo("Graph is not captured by backend if test with dynamo")
+    @skipIfCrossRef  # Arg order changes with crossref, see _crossref version of the test below
```

Comment:
There's no more crossref version of the test, right?

Reply:
Correct I removed it

Reply:
Can we fix the comment to not point to "the crossref version of the test"